mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -913,6 +929,13 @@ static LogicalResult printOperation(CppEmitter &emitter,
 
   os << "(";
   Operation *operation = functionOp.getOperation();
+  if (functionOp.isExternal()) {
```

Comment:
Use `functionOp.isDeclaration()`.

Reply:
See discussion in reply to your overall comment.